portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase

```diff
@@ -157,6 +157,74 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-139101. Verify missing references checkbox only prevents validation during Site import processes, not portlet import processes."
```

Comment:
The expected behavior is that missing references validation also works on portlet import processes.
Your description is the bug description.